src/services/ajv.ts

```diff
@@ -70,10 +71,18 @@ function defaultResourceSchema(resourceSchema: any, collection: string) {
 
     if (collectionNameFieldKey) {
         // Add a default property set to the stripped collection name
-        const modifiedSchema = resourceSchema;
-
-        modifiedSchema.properties[collectionNameFieldKey].default =
-            stripPathing(collection);
+        let modifiedSchema = {};
```

Comment:
What is driving the use of `let` here?

Reply:
Working through the kinks of the bug you noted, I reassigned this variable. Agreed it can become a constant now.